introduction.md

```diff
@@ -0,0 +1,3 @@
+<<<<<<< HEAD
```

Comment:
Of course, the GitHub path is good, but I have also enjoyed Python (and Django) and Database paths and also done some Electives and HTML work.